redis/src/connection.rs

```diff
@@ -979,6 +983,20 @@ fn setup_connection(
         }
     }
 
+    if connection_info.client_name.is_some() {
```

Comment:
same - `if let` and `?`

Reply:
link to an example

Reply:
will do

Reply:
this non async version of setup connection uses is_some() with the password, will stick to that